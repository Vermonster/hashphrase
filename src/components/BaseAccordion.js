import React from 'react';
import { View } from 'react-native';
import { List, Card, Paragraph } from 'react-native-paper';

const BaseAccordion = (props) => {
  const { title, contents } = props;
  return (
    <View>
      <List.Accordion title={title}>
        <Card>
          <Card.Content>
            <Paragraph>
              {contents}
            </Paragraph>
          </Card.Content>
        </Card>
      </List.Accordion>
    </View>
  );
};

export default BaseAccordion;
