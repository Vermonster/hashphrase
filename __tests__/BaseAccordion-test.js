import React from 'react';
import { shallow } from 'enzyme';
import BaseAccordion from '../src/components/BaseAccordion';

describe('<TipsPage />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <BaseAccordion title="test title" contents="test contents" />,
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
