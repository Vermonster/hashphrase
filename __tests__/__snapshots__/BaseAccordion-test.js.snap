// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TipsPage /> should match the snapshot 1`] = `
<ForwardRef(View)>
  <withTheme(List.Accordion)
    title="test title"
  >
    <withTheme(Card)>
      <Card.Content>
        <Paragraph>
          test contents
        </Paragraph>
      </Card.Content>
    </withTheme(Card)>
  </withTheme(List.Accordion)>
</ForwardRef(View)>
`;
